homeassistant/components/androidtv/strings.json

```diff
@@ -32,12 +32,12 @@
         "title": "Android TV Options",
         "data": {
           "apps": "Configure applications list",
-          "get_sources": "Whether or not to retrieve the running apps as the list of sources",
-          "exclude_unnamed_apps": "Exclude app with unknown name",
-          "screencap": "Determines if album art should be pulled from what is shown on screen",
+          "get_sources": "Retrieve the running apps as the list of sources",
+          "exclude_unnamed_apps": "Exclude apps with unknown name",
+          "screencap": "Use screen capture for album art",
           "state_detection_rules": "Configure state detection rules",
-          "turn_off_command": "ADB shell command to override default turn_off command",
-          "turn_on_command": "ADB shell command to override default turn_on command"
+          "turn_off_command": "Turn off command (leave empty for default)",
+          "turn_on_command": "Turn on command (leave empty for default)"
```

Comment:
I think it's good to mention that these are ADB shell commands, not HA commands (or some other kind of command).